meson.build

```diff
@@ -276,4 +276,9 @@ if get_option('apps')
 endif
 
 pkg = import('pkgconfig')
-pkg.generate(nanoarrow_lib)
+pkg.generate(libraries : nanoarrow_lib,
+             subdirs : [incdir],
+             version : meson.project_version(),
+             name : 'Apache Arrow nanoarrow',
```

Comment:
The default value without for this is `nanoarrow`. I think it should stay that but not sure how the branding works - @paleolimbot would know best